dm/ctl/master/start_task.go

```diff
@@ -23,15 +23,17 @@ import (
 	"github.com/pingcap/dm/checker"
 	"github.com/pingcap/dm/dm/ctl/common"
 	"github.com/pingcap/dm/dm/pb"
+	"github.com/pingcap/dm/pkg/terror"
 )
 
 // NewStartTaskCmd creates a StartTask command
 func NewStartTaskCmd() *cobra.Command {
 	cmd := &cobra.Command{
-		Use:   "start-task [-s source ...] <config-file>",
+		Use:   "start-task [-s source ...] <config-file> [--remove-meta]",
```

Comment:
How about putting it after `[-s source ...]` (before `<config-file>`, to make flags closer)? if you accept this comment, do not forget to update relative test cases.

Reply:
addressed